packages/app-studio-toolkit/src/apis/isOpenedForAction.ts

```diff
@@ -0,0 +1,24 @@
+import { getLogger } from "../logger/logger";
+import { getParameter } from "./parameters";
+
+export async function isOpenedForAction(): Promise<boolean | undefined> {
+  const logger = getLogger().getChildLogger({ label: "isOpenedForAction" });
+
+  const pkgActionParam = await getParameter("pkg-action");
+  if (pkgActionParam === undefined) {
+    logger.trace(`Package action does not exist!`);
+    return undefined;
+  }
+  if (
+    pkgActionParam === "release" ||
```

Comment:
instead of equality to several strings
you could try lodash's includes/contains on an array of valid values